src/sentry/sentry_apps/web/debug_sentry_app_notify_disable.py

```diff
@@ -7,7 +7,7 @@
 from sentry.sentry_apps.models.sentry_app import SentryApp
 from sentry.sentry_apps.models.sentry_app_installation import SentryAppInstallation
 
-from .mail import MailPreview
+from ...web.frontend.debug.mail import MailPreview
```

Comment:
```suggestion
from sentry.web.frontend.debug.mail import MailPreview
```

Could we use absolute imports instead of deep relative ones?

Reply:
yes thank you for catching that 🙇‍♀️